content/legacy/6_gard.md

```diff
@@ -0,0 +1,11 @@
+---
+image: legacy/gard.png
+headline: Gard, France
+---
+
+## Gard, France, in September 2002
+
+
+This event is documented in detail by Delrieu et al., 2005. A landfalling mesoscale convective system remained stationary for almost two days, trapped by the orography. It brough horrendous amounts of rainfall and caused wide-spread flooding.
```

Comment:
Can we make the references link to the corresponding doi's?

Reply:
Done!